packages/balances-react/src/atoms/balances.ts

```diff
@@ -181,15 +181,14 @@ const balancesSubscriptionAtomEffect = atomEffect((get) => {
         addressesByTokenByModule[token.type][token.id] = allAddresses.filter((address) => {
           // for each address, fetch balances only from compatible chains
           return isEthereumAddress(address)
-            ? !!token.evmNetwork?.id || chainsById[token.chain?.id ?? ""]?.account === "secp256k1"
-            : !!token.chain?.id
+            ? token.evmNetwork?.id || chainsById[token.chain?.id ?? ""]?.account === "secp256k1"
+            : token.chain?.id && chainsById[token.chain?.id ?? ""]?.account !== "secp256k1"
```

Comment:
New exclusion: don't try to fetch account balances for SS58-style accounts on moonbeam/moonriver